server/db/clean-database-backup.sql

```diff
@@ -0,0 +1,27 @@
+-- sql for purging pools and its' related entities.
+-- helpful for database backups where ExpiredPoolsJob has not run for a long time.
+SET FOREIGN_KEY_CHECKS=0;
+
+DELETE FROM cp_pool_attribute WHERE pool_id = (SELECT id FROM cp_pool WHERE enddate < now() AND id = cp_pool_attribute.pool_id);
```

Comment:
@Ceiu should be okay if the intent of this script is just cleanup the db for testing , yes?